src/components/plots/helpers/migrateFeaturesVsUMIsPlotData.jsx

```diff
@@ -0,0 +1,27 @@
+const migrateFeaturesVsUMIsPlotData = (plotData) => {
```

Comment:
Maybe change to something more descriptive? something along `transformOldDataToDataNewData`